README.md

```diff
@@ -56,7 +56,7 @@
 | [SPIRV-Tools](https://github.com/KhronosGroup/SPIRV-Tools) | `CMake` | `clvk` dependencies |
 | [SPIRV-Headers](https://github.com/KhronosGroup/SPIRV-Headers) | `CMake` | `clspv` & `SPIRV-Tools` dependencies |
 | [angle](https://github.com/google/angle) (WIP) | `GN` | `libEGL_angle.so`, `libGLESv2_angle.so`, `libGLESv1_CM_angle.so` |
-| [mesa](https://www.mesa3d.org/) | `meson` | `libEGL_mesa.so`, `libGLESv2_mesa.so`, `libGLESv1_CM_mesa.so`, `libvulkan.so` |
+| [mesa](https://www.mesa3d.org/) | `meson` | `libgallium_dri.so`, `libglapi.so`, `libEGL_mesa.so`, `libGLESv2_mesa.so`, `libGLESv1_CM_mesa.so`, `libvulkan_intel.so` |
```

Comment:
This line is not Intel specific even so we only have the Intel flavor for it here for now.
```suggestion
| [mesa](https://www.mesa3d.org/) | `meson` | `libgallium_dri.so`, `libglapi.so`, `libEGL_mesa.so`, `libGLESv2_mesa.so`, `libGLESv1_CM_mesa.so`, `libvulkan.so` |
```

Reply:
That is true, but `libvulkan.so` is a file provided by the Vulkan loader, while `libvulkan_${VENDOR}.so` is provided by the vendor's driver. Maybe I should change it to `libvulkan_${VENDOR}.so` instead?

Reply:
sounds good to me